playground/mocks/data/idp/idx/terminal-enduser-email-consent-denied.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This is the remediation SIW gets in the first device after the user denied signin.